rebench/tests/persistency_test.py

```diff
@@ -168,3 +174,58 @@ def _exec_rebench_db(self, cmd_config, server):
 
         run = list(cnf.get_runs())[0]
         run.close_files()
+
+    def test_check_file_lines(self):
+        ds = DataStore(self.ui)
+        cnf = Configurator(load_config(self._path + '/persistency.conf'),
+        ds, self.ui, data_file=self._tmp_file)
```

Comment:
I am a bit surprised that Python does parse this.
Could you indent this line to line up with the `load_config` in the line above?
Same as for instance `test_iteration_invocation_semantics()`